### DISCUSSION TITLE
Feature request: show time zone offset in profile card

### DISCUSSION THREAD

**Discussion by heaths** [community/community#102931] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

When interacting with contributors or maintainers, it can be helpful to know - within reasonable expectations - how long till they might see it. Specifically, should I write up a chunky reply with lots of different options because they are half a world away time-wise (assuming core hours), or could I maybe expect a more chatty back-and-forth because they are closer to my time zone?

Certainly not everyone works core hours or might even reply within a day or two, but when there's active engagement on an issue or PR, having this knowledge could help craft replies. I and several colleagues use this information in Teams, for example, to craft chunky or chatty replies based on time-based proximity.

<img width="253" alt="image" src="https://github.com/community/community/assets/1532486/589fe575-99c3-44b9-aca1-0011c3cbdcf5">

Already, there is a time zone setting profile next to Location that could be used:

![image](https://github.com/community/community/assets/1532486/dd672ce7-b0e8-4482-b13a-261c4f5f4f32)

In the profile card, showing something like "8 hours ahead" could help me be more productive in my interaction. The Teams pic also shows that they can enter core work hours (in Teams settings, and in Outlook settings I believe as well) and maybe that's useful, but given many involved in OSS have work and off-work hours in the same profile, perhaps not that helpful here. Still, showing the offset could be very useful.

I envision this in an easily- and quickly-accessed place like the profile card that appears when you hover over someone's handle. It could also be in the profile, but not having to click away to see it would be ideal at least.

**Comment by williammartin** [comment, marked as answer]
@heaths I think maybe this exists but when you were hovering my card you didn't see it because I didn't have it added to my profile. I've added it now, let me know what you see. Here's what I see for you: 

<img width="447" alt="image" src="https://github.com/community/community/assets/1611510/42df8804-e309-429f-bd52-dab525e87623">

**Reply by heaths** [reply]
Interesting. I tried a few others before I opened this, including colleagues at work. Seems few people have it set. Odd, given it shows you timestamps in your local time - the main reason I set it.

I don't recall if that was way back when I opened my account in 2008 or if that was added later. If the latter, perhaps a way to advertise new features or settings would be good. As unobtrusive as showing a little colored circle on menus for new things, to cards that point out new features in seeing in more and more apps (I prefer the little colored circle overlay, but it is less obvious).

Just a thought. I'll share with my colleagues that seemed interested in the idea. Guess they didn't have it set.
